### PR TITLE
Handle top level TOML series definitions

### PR DESCRIPTION
**Pull request by jestapinski**
This PR supports the top-level TOML means of defining a series by adding some build-time logic to the `createPages` step. Given data that can look like:

```
    const allSeries = {
        'test-articles': [
            'article/article',
            'quickstart/nodejs-how-to-connect-to-your-database',
        ],
        'test-articles-two': [
            'article/active-active-application-architectures-with-mongodb',
            'quickstart/nodejs-how-to-connect-to-your-database',
            'how-to/working-with-mongodb-stitch-through-the-mongo-shell',
        ],
    };
```

this PR handles attaching relevant data about each series an article is in to its context at build-time.

I also added support for articles in multiple series, and updated the stylings for series a bit to match updated styles.

Screen Shot 2020-03-02 at 11 30 59 AM

### CODE REVIEW

File: tests/unit/get-series-articles.test.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import { getSeriesArticles } from '../../src/utils/get-series-articles';
+
+it('should correctly determine which series an article belongs to', () => {
```
Comment (jestapinski) [inline tests/unit/get-series-articles.test.js]: I wanted to write a test to best document logic, but didn't know where the best place to put it was. This directory was already configured for just so I put it here.

File: src/components/dev-hub/series.js [hunk 2]
```diff
@@ -34,8 +34,8 @@ const activeLiStyles = css`
     font-size: 22px;
     &:after {
         /* Adjust the background bullet position for the bull's eye text */
-        font-size: 38px;
-        top: -2px;
+        font-size: 54px;
+        top: 1px;
```
Comment (greggb) [inline src/components/dev-hub/series.js]: I think the active bullet needs to drop about 1-2px lower to be even with the text

File: src/components/dev-hub/series.js [hunk 3]
```diff
@@ -146,7 +146,7 @@ const SeriesList = styled('ul')`
             ),
             ${BORDER_GRADIENT};
         background-size: ${size.medium} 2px, cover;
-        bottom: ${BULLET_SIZE / 2}px;
+        bottom: calc(${BULLET_SIZE / 2}px + 2px);
```
Comment (greggb) [inline src/components/dev-hub/series.js]:
```suggestion
        bottom: ${(BULLET_SIZE / 2)+2}px);
```

File: src/utils/get-series-articles.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
+// given a page slug, find all occurrances in "allSeries"
+const getSeriesArticles = (allSeries, slug) => {
+    if (!allSeries || !slug) return [];
+    const series = Object.keys(allSeries);
+    const seriesForThisArticle = {};
+    series.forEach(s => {
+        if (allSeries[s].includes(slug)) seriesForThisArticle[s] = allSeries[s];
```
Comment (greggb) [inline src/utils/get-series-articles.js]:
```suggestion
        if (allSeries[s].includes(slug)) {
            seriesForThisArticle[s] = allSeries[s] 
        };
```

**Comment by jestapinski** [comment]
Thanks!

**Pull request merged**